sdks/java/io/hbase/pom.xml

```diff
@@ -52,29 +50,6 @@
               </goals>
               <configuration>
                 <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
I am not sure if we need to shade protobuf anymore since the external API uses the same version than Beam (only for ByteString), and the internal API (that on the HBase client is now using the shaded dependency).